helm/values.schema.json

```diff
@@ -0,0 +1,1856 @@
+                              "type": "string",
+                              "title": "The trust root's name schema",
+                              "description": "Name of the trust root",
+                              "examples": [
+                                "mykey"
+                              ]
+                            },
+                            "key": {
+                              "$id": "#/properties/validators/items/anyOf/2/properties/trust_roots/anyOf/0/properties/key",
+                              "type": "string",
+                              "title": "The trust root's key schema.",
+                              "pattern": "(?:-+BEGIN\\sPUBLIC\\sKEY[-]+)\n(?:(?:[A-Za-z0-9+\\/\\s])*={0,2})\n(?:-+END\\sPUBLIC\\sKEY[-]+)",
+                              "description": "Public key used for verification of signatures on container images whose policy matches this trust root.",
+                              "examples": [
+                                "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEvtc/qpHtx7iUUj+rRHR99a8mnGni\nqiGkmUb9YpWWTS4YwlvwdmMDiGzcsHiDOYz6f88u2hCRF5GUCvyiZAKrsA==\n-----END PUBLIC KEY-----\n"
+                              ]
+                            }
+                          },
+                          "required": [
+                            "name",
+                            "key"
+                          ],
+                          "minItems": 1,
+                          "uniqueItems": true
+                        }
+                      ]
+                    },
+                    "required": [
+                      "trust_roots"
+                    ]
+                  },
+                  "cert": {
+                    "$id": "#/properties/validators/items/anyOf/2/properties/cert",
+                    "title": "The cert schema",
+                    "description": "TLS certificate if using a registry requiring a self-signed certificate.",
+                    "type": "string",
+                    "pattern": "(?:-+BEGIN\\sCERTIFICATE[-]+)\n(?:(?:[A-Za-z0-9+\\/\\s])*={0,2})\n(?:-+END\\sCERTIFICATE[-]+)"
+                  },
+                  "host": {
+                    "$id": "#/properties/validators/items/anyOf/2/properties/host",
+                    "title": "The host schema",
+                    "description": "Not yet implemented.",
+                    "type": "string",
+                    "pattern": "(([-a-zA-Z0-9@:%._\\+~#=]{1,256}\\.[a-zA-Z0-9()]{1,6})|(localhost))(\\:\\d{2,5})?"
+                  },
+                  "auth": {
+                    "type": "object",
+                    "$id": "#/properties/validators/items/anyOf/2/properties/auth",
+                    "title": "The auth schema",
+                    "description": "Authentication credentials if using a private registry instance.",
+                    "oneOf": [
```

Comment:
I can't seem to break it if I set both or set k8s_keychain to a string

Reply:
Should work now :+1: Thanks for the hint!